src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+//import './index.css';
```

Comment:
Kullanılmayacaksa doğrudan kaldırılabilir.

Reply:
Kullanma konusunda karasız olduğum için yorum satırına ekledim. Ama gerek olmayacak gibi